### PR TITLE
Update build badges in README

### PR DESCRIPTION
**Pull request by GMNGeoffrey**

### CODE REVIEW

**Comment by GMNGeoffrey** [comment]
The cmake-bazel links are currently broken because I'm fixing the path for those (it currently doesn't match the directory path like everything else does

File: README.md [hunk 1]
```diff
@@ -35,16 +35,23 @@ of feedback on any [communication channels](#communication-channels)!
...
+
+CI System | Build System  | Platform   | Architecture       | Configuration / Component    | Status
+:-------: | :-----------: | :--------: | :----------------: | :--------------------------: | :----:
+Kokoro    | Bazel         | Linux      | x86-64             |                              | [![kokoro_status_bazel/linux/x86-swiftshader/core](https://storage.googleapis.com/iree-oss-build-badges/bazel/linux/x86-swiftshader/core/main_status.svg)](https://storage.googleapis.com/iree-oss-build-badges/cmake-bazel/linux/x86-swiftshader/main_result.html)
+Kokoro    | CMake & Bazel | Linux      | x86-64-swiftshader | Integrations                 | [![kokoro status cmake-bazel/linux/x86-swiftshader](https://storage.googleapis.com/iree-oss-build-badges/cmake-bazel/linux/x86-swiftshader/main_status.svg)](https://storage.googleapis.com/iree-oss-build-badges/cmake-bazel/linux/x86-swiftshader/main_result.html)
```
Comment (ScottTodd) [inline README.md:42]: Ah, I guess I should have noticed this before / on your other change, but putting "swiftshader" and "turing" as part of the architecture is a bit weird :P

How do you feel about `x64-64 (SwiftShader)`?

**Review by ScottTodd — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (GMNGeoffrey) [reply README.md:42]: Hmmm but Turing is a GPU architecture...

https://en.wikipedia.org/wiki/Turing_(microarchitecture)

Comment (ScottTodd) [reply README.md:42]: Sure, but this is still conflating "CPU architecture" with either "GPU architecture" or "GPU software driver" :P 

(I don't care that much either way)

Comment (GMNGeoffrey) [reply README.md:42]: Ok fair, fair. I'll change it. We have too many columns already :-P So many different variables!

**Pull request merged**